packages/design-tokens/src/scss/generated/_colors.scss

```diff
@@ -1,171 +1,184 @@
 // Generated Colors from Supernova. Do not edit manually.
-$text-primary-default: #132930 !default;
-$text-secondary-default: #90a2a7 !default;
-$emotion-informative-default: #90cff1 !default;
-$emotion-informative-hover: #85c1e1 !default;
-$emotion-informative-active: #78a9c5 !default;
-$emotion-informative-disabled: #c4c4c4 !default;
-$emotion-informative-background: #ebf3f8 !default;
-$emotion-success-default: #b7d16d !default;
-$emotion-success-hover: #9ebf40 !default;
-$emotion-success-active: #8fad3a !default;
-$emotion-success-disabled: #c4c4c4 !default;
-$emotion-success-background: #f2f5e7 !default;
-$emotion-warning-default: #f1d07c !default;
-$emotion-warning-hover: #dfbb5f !default;
-$emotion-warning-active: #c29132 !default;
-$emotion-warning-disabled: #c4c4c4 !default;
-$emotion-warning-background: #f8f2e4 !default;
-$emotion-danger-default: #db5f7b !default;
-$emotion-danger-hover: #c95771 !default;
-$emotion-danger-active: #af4c62 !default;
-$emotion-danger-disabled: #c4c4c4 !default;
-$emotion-danger-background: #fcf1f4 !default;
-$action-link-primary-default: #29616f !default;
-$action-link-primary-hover: #1b5260 !default;
+$action-inverted-active: #d4d4d4 !default;
```

Comment:
I think supernova orders items by update time. So I added alphabetic order to the exporter. So now it looks like a lot of changes, but actually, I think only `border-colors` and `focus-colors` are modified or newly defined.

Reply:
just fyi

Reply:
The borders update is BC btw.